scripts/preprocessing/_05a_apply_ica.py

```diff
@@ -93,7 +93,10 @@ def apply_ica(*, cfg, subject, session):
     msg = 'Saving reconstructed epochs after ICA.'
     logger.info(**gen_log_kwargs(message=msg, subject=subject,
                                  session=session))
-    epochs_cleaned.save(fname_epo_out, overwrite=True, split_naming='bids')
+    epochs_cleaned.save(
+        fname_epo_out, overwrite=True, split_naming='bids',
+        split_size=cfg._epochs_split_size)
+    # _update_for_splits(out_files, 'epochs_cleaned')
```

Comment:
to cleanup

Reply:
These were intentionally left in to remind us to do it when working on caching :)